core/src/main/java/org/apache/accumulo/core/client/admin/NamespaceOperations.java

```diff
@@ -202,9 +203,9 @@ default Iterable<Entry<String,String>> getProperties(String namespace)
   }
 
   /**
-   * Gets a merged view of the properties of a namespace, which are inherited by tables in this
-   * namespace. Note that recently changed properties may not be available immediately. This new
-   * method returns a Map instead of an Iterable.
+   * Gets a merged view of the properties of a namespace from its parent configuration. These
+   * properties are inherited by tables in this namespace. Note that recently changed properties may
+   * not be available immediately. This new method returns a Map instead of an Iterable.
```

Comment:
I don't feel that the `This new method...` lines should be included.  They would only be valid for one version, until the are no longer "new".  IDE's will highlight the return type when someone tries to use it.  Feel free to ignore if it does seem useful and I'm missing that context.

Reply:
Yes I agree. I found it strange that "new" was included. It was already present and I didn't want to make any unnecessary changes. I'll remove it